main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -165,6 +170,22 @@ HistoryPanel.prototype._render = function() {
   this.resize();
 };
 
+HistoryPanel.prototype.showHideHistoryFacetSplash = function(clicked) {
+  var hideHistoryFacetSplash = JSON.parse(Refine.getPreference("ui.browsing.hideHistoryFacetSplash", false));
```

Comment:
```suggestion
  var hideHistorySplash = JSON.parse(Refine.getPreference("ui.browsing.hideHistoryFacetSplash", false));
```
I understand that you what to create ONE preference for the 2 splash, FacetSplash and HistorySplash? But it would be incoherent with the named used for the facetSplash… (`ui.browsing.hideBrowsingFacetSplash`).

You got to make a choice. ;-)